src/apis/MessagesEndpoint.es6.js

```diff
@@ -23,44 +25,83 @@ const parseGetBody = (res, apiResponse, apiRequest) => {
     c.data.replies = replies;
     apiResponse.addResult(TYPE_MAP[c.kind].fromJSON(c.data));
   });
+
+  const { before, after } = body.data;
+  apiResponse.meta = { before, after };
+
+  return apiResponse;
 };
 
-const parsePostBody = (res, apiResponse, apiRequest) => {
-  const { body } = res;
-  console.log(body);
+const parsePostBody = (apiResponse) => {
+  const { body } = apiResponse.response;
+  if (has(body, 'json.data.errors') && body.json.errors.length) {
+    // There's a problem -- return the errors
+    return { errors: body.json.errors };
```

Comment:
After the discussion in the editing post/comment patch, this should throw a `ValidationError`.